src/components/TermAccordion.tsx

```diff
@@ -78,27 +80,22 @@ export const TermAccordion: React.FC<TermAccordionProps> = (props) => {
         <AccordionSummary expandIcon={<ExpandIcon />}>
           <Box>
             {props.showVocabulary && (
-              <IriLabel
-                iri={props.term.vocabulary}
-                variant="body2"
-                color="textSecondary"
-              />
```

Comment:
We could delete this component, since no other component is using it. The original intent of this component was to hide the fetching logic of the name of the particular resource (e.g. vocabulary)

Reply:
If it is really the last occurrence then I'll delete it.

Reply:
I have found one dependency on the file - VocabularyLabel component. Not sure what's the plan with that, leaving IriLabel in place for the moment.

Reply:
I see. I missed that one. It seems like it is yet another dead component. I will fix this technical debt after this PR.